webapp/packages/core-events/src/NotificationService.ts

```diff
@@ -87,16 +88,21 @@ export class NotificationService {
     this.notify(notification, ENotificationType.Info);
   }
 
+  logSuccessInfo<T>(notification: INotificationOptions<T>): void {
+    this.notify(notification, ENotificationType.Success);
+  }
+
```

Comment:
there can't be success info, it's must be different types of notification it can be success, informative, or error